.github/workflows/android-e2e-test.yml

```diff
@@ -51,20 +57,25 @@ jobs:
       - name: Build app
         working-directory: ${{ env.WORKING_DIRECTORY }}
         run: yarn build-e2e-android
+      - name: Enable KVM
+        run: |
+          echo 'KERNEL=="kvm", GROUP="kvm", MODE="0666", OPTIONS+="static_node=kvm"' | sudo tee /etc/udev/rules.d/99-kvm4all.rules
+          sudo udevadm control --reload-rules
+          sudo udevadm trigger --name-match=kvm
```

Comment:
It would be best to understand why `kvm` is needed here and what do we do here 😄 

But if it works I guess we're just fine w/o it 🤷🏻‍♂️

Reply:
Testing the workflow without it right now 😄

Reply:
It is used by [android-emulator-runner](https://github.com/reactivecircus/android-emulator-runner?tab=readme-ov-file#running-hardware-accelerated-emulators-on-linux-runners) and it's necessary. I'm bringing it back

Reply:
Great find!

Reply:
What about the caches I asked in the main message?

Reply:
In my case AVD cache helped to shave off 1-2 minutes when I boot the emulator. Curious to see the result in your project 👀